site/layouts/partials/menu.html

```diff
@@ -0,0 +1,32 @@
+<nav class="menu">
+  <ul class="menu__inner menu__inner--desktop">
+    {{ range $.Site.Menus.main }}
+      {{ if not .HasChildren }}
+        <li><a href="{{ .URL }}">{{ .Name }}</a></li>
+      <!-- The default theme supports a drop-down for showMenuItems > 0. We re-use this class
+           for child menus instead. -->
+      {{ else }}
+        <ul class="menu__sub-inner">
+          <li class="menu__sub-inner-more-trigger">
+            {{ .Name }}
+            <span class="menu__sub-inner-more-trigger-icon">{{ partial "greater-icon.html" . }}</span>
+          </li>
+          <ul class="menu__sub-inner-more hidden">
+            {{ range .Children }}
+              <li><a href="{{ .URL }}">{{ .Name }}</a></li>
+            {{ end }}
+          </ul>
+        </ul>
+      {{ end }}
+    {{ end }}
+  </ul>
+
+  <!-- This is the theme default layouts/partials/menu.html -->
+  <ul class="menu__inner menu__inner--mobile">
```

Comment:
there was no CSS for mobile submenus, so I decided to let this ride. another way could be to concatenate when children exist (ex About/GitHub About/History, etc)